waffle-doppelganger/test/helpers/interfaces/Cap.json

```diff
@@ -0,0 +1,69 @@
+{
```

Comment:
We should not put json files to repository and build them instead as a part of build and/or test process, see [buildTestContracts.ts](https://github.com/EthWorks/Waffle/blob/fc645edd76024593ca97a80f551a6a595bea311d/waffle-cli/buildTestContracts.ts) for inspiration